mem/mem.c

```diff
@@ -384,6 +377,7 @@ int comdb2ma_init(size_t init_sz, size_t max_cap)
              * 3. If a thread is not allowed to create a mspace, it will
              *    be uniformly assigned one (to minimize contention).
              */
+            char *env_mspace_max;
```

Comment:
What compiler warning do you see here?

Reply:
it's an unused variable warning -- only used if PER_THREAD_MALLOC is defined.

Reply:
Gotcha.